templates/kubernetes/2/docker-compose.yml

```diff
@@ -44,47 +44,30 @@ proxy:
         - kubernetes:master
 
 etcd:
-  image: rancher/etcd:v2.3.6
+  image: llparse/etcd:v2.3.6-2
   labels:
-    io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
-    io.rancher.sidekicks: etcd-data
-    io.rancher.service.allocate.skip.serialize: 'true'
+    io.rancher.scheduler.affinity:container_label_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
     io.rancher.container.start_once: 'true'
+    io.rancher.sidekicks: data
   environment:
     ETCD_DATA_DIR: /data
     ETCDCTL_ENDPOINT: http://etcd:2379
+    RANCHER_DEBUG: ${DEBUG}
   links:
-  - etcd-data
+  - data
   - discovery
   volumes_from:
-  - etcd-data
+    - data
 
-etcd-data:
-  image: busybox
-  command: /bin/true
+data:
+  image: llparse/etcd
```

Comment:
You know what? I don't see enough value in the :latest hack to warrant changing from busybox. I'm going to change it back.

I'll build the rancher/etcd:v2.3.6-2 and update this PR along with that change.